apps/gitness/src/components-v2/breadcrumbs/project-dropdown.tsx

```diff
@@ -17,7 +19,24 @@ function ProjectDropdown(): JSX.Element {
   const routes = useRoutes()
   const { spaceId } = useParams<PathParams>()
   const navigate = useNavigate()
-  const { spaces } = useAppContext()
+  const { spaces, setSpaces } = useAppContext()
+
+  useEffect(() => {
+    if (!spaces?.length) {
+      membershipSpaces({
+        queryParams: { page: 1, limit: 100, sort: 'identifier', order: 'asc' }
+      })
+        .then(({ body: memberships }) => {
+          const spaceList = memberships.filter(item => item?.space).map(item => item.space as TypesSpace)
+          setSpaces(spaceList)
+        })
+        .catch(() => {
```

Comment:
all errors should not trigger a navigation to signin. i think that is already handled via global 401 handling.
here we can simple disable the dropdown maybe for now.

Reply:
Done.